src/fragmentarium/ui/display/DisplayLineNumbers.tsx

```diff
@@ -0,0 +1,41 @@
+import { Line, LineNumber, LineNumberRange, TextLine } from '../../domain/text'
+import React from 'react'
+
+function formatLineNumber(lineNumber: LineNumber): string {
+  return `${lineNumberToString(lineNumber)}`
+}
+
+function formatLineNumberRange({ start, end }: LineNumberRange): string {
+  return `${lineNumberToString(start)}-${lineNumberToString(end)}`
+}
+function lineNumberToString({
+  hasPrime,
+  number,
+  prefixModifier,
+  suffixModifier
+}: LineNumber): string {
+  const prefixMod = prefixModifier ? prefixModifier + '+' : ''
+  const prime = hasPrime ? '′' : ''
+  const suffixMod = suffixModifier ? suffixModifier : ''
+  return `${prefixMod}${number}${prime}${suffixMod}`
+}
+
+const lineNumberTypeToString = {
+  LineNumber: formatLineNumber,
+  LineNumberRange: formatLineNumberRange
+}
+function chooseLineNumberType(
```

Comment:
The name is misleading. Choosing is an implementation detail which should not concern the caller. The function converts the line number to a string.